tokio/src/time/instant.rs

```diff
@@ -67,13 +67,10 @@ impl Instant {
         self.std
     }
 
-    /// Returns the amount of time elapsed from another instant to this one.
-    ///
-    /// # Panics
-    ///
-    /// This function will panic if `earlier` is later than `self`.
```

Comment:
is it maybe worth explicitly noting somewhere that "unlike the `std::time::Instant` equivalent, this function does not panic" or similar? not a blocker.